plugins/inputs/webhooks/plex/plex_webhooks_test.go

```diff
@@ -0,0 +1,25 @@
+package plex
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+)
+
+func PlexWebhookRequest(jsonString string, t *testing.T) {
+	var acc testutil.Accumulator
+	gh := &PlexWebhook{Path: "/plex", acc: &acc}
+	req, _ := http.NewRequest("POST", "/plex", strings.NewReader(jsonString))
+	w := httptest.NewRecorder()
+	gh.eventHandler(w, req)
+	if w.Code != http.StatusOK {
```

Comment:
Telegraf's current preferred testing practice is to use `stretchr/testify/require` for tests. `require.Eqaul(t, http.StatusOk, w.Code)`